src/twisted/names/test/test_names.py

```diff
@@ -469,6 +502,66 @@ def test_zoneTransferConnectionFails(self):
             resolver.lookupZone("impossible.invalid"), error.DNSLookupError
         )
 
+    def _testHelper_zoneTransferFailure(
```

Comment:
This should be `_testHelperZoneTransferFailure` because it is not dynamically dispatched.